src/noteburst/main.py

```diff
@@ -30,23 +30,20 @@
     name=config.logger_name,
 )
 
-app = FastAPI()
-"""The main FastAPI application for noteburst."""
-
-# Define the external routes in a subapp so that it will serve its own OpenAPI
-# interface definition and documentation URLs under the external URL.
-external_app = FastAPI(
-    title="noteburst",
-    description=metadata("noteburst").get("Summary", ""),
+app = FastAPI(
+    title=config.name,
+    description=Path(__file__).parent.joinpath("description.md").read_text(),
```

Comment:
I decided I wanted a longer description for the Open API docs where I could use Markdown formatting and the package summary didn't fit the bill for this. Potentially I could have instead used the python package description that's generated from the package README. Still, this is an awkward solution because you're making a README fit both an introduction to a source repo and an introduction to a web API. So at the end of the day I decided to write an introduction that's specifically for the Open API docs.